scopes/mdx/modules/compiler/compile-output.ts

```diff
@@ -22,6 +22,14 @@ export class CompileOutput {
     return data.frontmatter;
   }
 
+  /**
+   * get all import specifiers.
+   */
+  getImportSpecifiers() {
```

Comment:
I'd add types here. 
does this function returns array of strings? if so, I wouldn't call it ImportSpecifer, it's just dependencies. The `importSpecifier` we use all over is what we get from the js/ts compilers as ImportSpecifier object.